cypress/e2e/course_management/showCoursesInTable.feature

```diff
@@ -0,0 +1,98 @@
+        Then I can see the course '<course_title>' without classes and with teacher '<teacher_name>'
+        Then I can see 3 enabled action items for course '<course_title>'
+        Examples:
+            | course_title               | teacher_name  |
+            | Cypress-Admin-Test-Course  | Karl Herzog   |
+#
+    Scenario Outline: As an admin i can edit the courses of my school
+        Given I see the new course administration page
+        When I click the edit button on the '<course_title>' course
+        Then I see page Edit course
+        Then I see the course title is '<course_title>'
+        When I click the cancel edit course button
+        Then I can see the cancel modal
+        When I click the confirmation button on the cancel modal
+        Then I see the new course administration page
+        When I click the edit button on the '<course_title>' course
+        Then I click on the save course changes button
+        Then I see the new course administration page
+    Examples:
+        | course_title               |
+        | Cypress-Admin-Test-Course  |
+#
+    Scenario Outline: As an admin i can synchronize a course with a group
+        Given I see the new course administration page
+        When I click the start synchronization button on the '<course_title>' course
+        Then I see title of the modal to select a synced group
+        Then I see information text of the modal to select a synced group
+        Then I see the group selection of the modal to select a synced group
+        When I select group '<group_title>' in the group selection
+        Then I see the group '<group_title>' is selected
+        When I click continue button on the modal to select a synced group
+        Then I see the title of the modal for synchronization confirmation
+        Then I see a warning about the consequences of synchronization
+        Then I see information text of the modal asking for confirmation of synchronization
+        When I click the confirm button on the synchronization confirmation modal
+        Then I see the new course administration page
+        #in future a stop synchronization button will be seen here and should be tested
+        Then I do not see the start synchronize button for '<course_title>' anymore
+    Examples:
+        | course_title               | group_title         |
+        | Cypress-Admin-Test-Course  | Cypress-Test-Group  |
+#
+    Scenario Outline: As an admin i can delete courses
+        Given I see the new course administration page
+        When I click the delete button for course '<course_title>' in course table
+        Then I can see the delete modal
+        When I click the confirmation button on the delete modal
+        Then I see the new course administration page
+    Examples:
+        |course_title               |
+        |Cypress-Admin-Test-Course  |
+#
+    Scenario: As a teacher i can not see the new course admin page, when the feature flag is off
```

Comment:
This is more of a sidebar test. And what feature flag allows teachers to see the course administration?